pkg/application/options.go

```diff
@@ -305,3 +319,18 @@ func WithUTCClock(useUTC bool) Option {
 		})
 	}
 }
+
+func WithVersion(version string) Option {
+	return func(app *App) {
+		app.addSetupOption(func(config cfg.GosoConf, logger mon.GosoLog) error {
```

Comment:
feels weird to use a setup option to log something..

Reply:
Please #BeDirect and let me know where you would put it

Reply:
well, i don't have always a better idea right away, but my first impulse would be something like a `-version` flag for your command..